code/obj/storage/loot_crates.dm

```diff
@@ -180,7 +180,7 @@
 					picker = rand(1,10)
 					switch(picker)
 						if(1)
-							items += pick(/obj/item/gun/energy/laser_gun/pred)
+							items += pick(/obj/item/gun/energy/laser_gun)
```

Comment:
is knocking this down to the regular one from an advanced gun on purpose?

Reply:
Whoops